flight_paths.py

```diff
@@ -0,0 +1,89 @@
+import json
+from simple_graph import Node, SimpleGraph
+from simple_graph import spt_Dijkstra
+
+
+def extract_data_from_json(file):
+    with open(file) as json_data:
+        data = json.load(json_data)
+    new_data = []
+    for d in data:
+        entry = {}
+        entry['city'] = d['city']
+        entry['neighbors'] = d['destination_cities']
+        entry['lat-lon'] = d['lat_lon']
+        new_data.append(entry)
+    return new_data
+
+
+def build_graph(new_data):
+    gr = SimpleGraph()
+
+    for city in new_data:
+        n = Node(city['city'], city['lat-lon'])
+        try:
+            gr.add_node(n)
+        except KeyError:
```

Comment:
It's when run with Python 2. You might want to consider turning everything into unicode literals for this assignment. I think it's a `from __future__` thing you can do. I forget right now. WHY ARE YOU AWAKE?!